docs/index.md

```diff
@@ -7,305 +7,46 @@ page_keywords: docker, swarm, clustering
 # Docker Swarm
 
 Docker Swarm is native clustering for Docker. It turns a pool of Docker hosts
-into a single, virtual host.
+into a single, virtual host. Because Docker Swarm serves the standard Docker
+API, any tool that already communicates with a Docker daemon can use Swarm to
+transparently scale to multiple hosts. Supported tools include, but are not  limited to, the following: 
 
-Swarm serves the standard Docker API, so any tool which already communicates
-with a Docker daemon can use Swarm to transparently scale to multiple hosts:
-Dokku, Compose, Krane, Deis, DockerUI, Shipyard, Drone, Jenkins... and,
-of course, the Docker client itself.
+- Dokku
+- Docker Compose
+- Krane
+- Jenkins
 
-Like other Docker projects, Swarm follows the "batteries included but removable"
-principle. It ships with a simple scheduling backend out of the box, and as
-initial development settles, an API will develop to enable pluggable backends.
-The goal is to provide a smooth out-of-box experience for simple use cases, and
-allow swapping in more powerful backends, like Mesos, for large scale production
-deployments.
+And of course, the Docker client itself is also supported.
 
-## Release Notes
+Like other Docker projects, Docker Swarm follows the "batteries included but
+removable" principle. It ships with a simple scheduling backend out of the box.
```

Comment:
How about:

> It ships with a simple default scheduling backend out of the box.

@vieux ?

I feel *simple* is minimizing the default scheduler.